### PR TITLE
Dropout

### PR DESCRIPTION
**Pull request by shakeebmajid**
dropout added in, seems to regularize somewhat but not too much with these current hyperparameters

### CODE REVIEW

**Review by brendanvonhofe — APPROVED** [review]
good stuff, ill give it try and see if it improves training
